hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/XceiverClientGrpc.java

```diff
@@ -382,10 +382,10 @@ private XceiverClientReply sendCommandWithRetry(
       } catch (IOException e) {
         ioException = e;
         responseProto = null;
-        LOG.debug("Failed to execute command {} on datanode {}",
+        LOG.error("Failed to execute command {} on datanode {}",
```

Comment:
This may lead of verbose logging on client logs/console.

Reply:
Will revert the log level change.